src/apps/content-editor/src/app/views/ItemCreate/ItemCreate.js

```diff
@@ -63,6 +65,10 @@ export default function ItemCreate() {
     }
   }, [modelZUID, item]);
 
+  useEffect(() => {
+    dispatch(actions.setDuoMode(false));
```

Comment:
won't this turn it off though? meaning anytime you create a new item you will be booted out of duomode and forced to toggle back on?

Reply:
@stuart This was the best solution I had in mind beside trying to display none a lot of elements while in Item create view, unless you have a better route?

Reply:
You will need to lift the DuoMode preview out of `Editor.js` and into https://github.com/zesty-io/manager-ui/blob/f6c5d14fd234848b9bf17ce8ebd13790bb18aad2/src/apps/content-editor/src/app/views/ItemEdit/Content/Content.js

Reply:
@shrunyan  I have reevaluated the styles and got duo mode to hide correctly for item create with out touching state.